contract/src/signature_verifier.cairo

```diff
@@ -0,0 +1,38 @@
+use starknet::eth_signature::{is_eth_signature_valid};
+use starknet::EthAddress;
+use starknet::secp256_trait::Signature;
+
+
+//| Account address |  0x30A333e4207129AD466a0B88b329651833605de4
+//| Private key     |  0x89ac8b3450f4f77fba1214fde42da15bc0d6b74cd1e77d8831528143ed8dc52c
+//| public key      |  0x02d8834cfd7957d3cdf4ddd4ff038e6f8c06f31170d486cb52b737c0d41b0273e0
+
+const public_key : felt252 =  0x30A333e4207129AD466a0B88b329651833605de4;
+
+fn verify_signature(commitment: u256, signature: Signature) {
```

Comment:
our doc standard is to have natural language paragraphs (generous term, anything between 1-3 sentences is sufficient) above every function call 

then comments within the functions are good whenever helpful, but not required